### PR TITLE
change inactive buttons look

### PR DESCRIPTION
**Pull request by joebrock666**

### CODE REVIEW

**Comment by bludnic** [comment]
@joebrock666 check the task https://trello.com/c/DFsBjI1k/333-ui-inactive-buttons

**Comment by joebrock666** [comment]
> @joebrock666 check the task https://trello.com/c/DFsBjI1k/333-ui-inactive-buttons

Done

File: src/assets/styles/generic/_vuetify.scss [hunk 1]
```diff
@@ -39,6 +39,21 @@ html {
   border-radius: 2px;
 }
 
+.v-theme--light {
+  .v-btn--disabled:not(.v-btn--icon):not(.v-btn--flat),  {
+    background-color: #eee; // secondary
```
Comment (bludnic) [inline src/assets/styles/generic/_vuetify.scss:44]: Use color scheme
map-get($adm-colors, 'color_name_here');

File: src/assets/styles/generic/_vuetify.scss [hunk 2]
```diff
@@ -39,6 +39,21 @@ html {
...
+    color: #bbdefb !important; // primary2
```
Comment (bludnic) [inline src/assets/styles/generic/_vuetify.scss:45]: map-get($adm-colors, 'color_name_here');

File: src/assets/styles/generic/_vuetify.scss [hunk 3]
```diff
@@ -39,6 +39,21 @@ html {
...
+    box-shadow: none !important;
+  }
+}
+.v-theme--dark {
+  .v-btn--disabled:not(.v-btn--icon):not(.v-btn--flat),  {
+    background-color: #4a4a4a !important; // regular
```
Comment (bludnic) [inline src/assets/styles/generic/_vuetify.scss:51]: map-get($adm-colors, 'color_name_here');

File: src/assets/styles/generic/_vuetify.scss [hunk 4]
```diff
@@ -39,6 +39,21 @@ html {
...
+    color: #9e9e9e !important; // grey
```
Comment (bludnic) [inline src/assets/styles/generic/_vuetify.scss:52]: map-get($adm-colors, 'color_name_here');

**Review by bludnic — CHANGES_REQUESTED** [review]

File: src/assets/styles/generic/_vuetify.scss [hunk 5]
```diff
@@ -39,6 +39,28 @@ html {
...
+    background-color: hsla(0, 0%, 100%, 0.35) !important;
+    color: hsla(0, 0%, 100%, 0.95) !important;
+    box-shadow: none !important;
+    opacity: 0.36;
+  }
+  .v-btn__overlay {
+    opacity: 0 !important;
+  }
+}
+
```
Comment (bludnic) [inline src/assets/styles/generic/_vuetify.scss:63]: The styles affects other buttons with `variant="text"`

image

You must apply these styles only for `variant="elevated"` which is by default.

See all variants https://vuetifyjs.com/en/api/v-btn/#props-variant

File: src/components/LoginPasswordForm.vue [hunk 6]
```diff
@@ -94,7 +92,6 @@ export default defineComponent({
```
Comment (bludnic) [inline src/components/LoginPasswordForm.vue:92]: `const isValidForm` is not used anymore. You can remove it

**Review by bludnic — CHANGES_REQUESTED** [review]

File: src/assets/styles/generic/_vuetify.scss [hunk 7]
```diff
@@ -40,12 +40,12 @@ html {
 }
 
 .v-theme--light {
-  .v-btn--disabled:not(.v-btn--icon):not(.v-btn--flat),  {
+  .v-btn--disabled:not(.v-btn--icon):not(.v-btn--flat).v-btn--variant-elevated  {
```
Comment (bludnic) [inline src/assets/styles/generic/_vuetify.scss:43]: The button cannot be `flat` and `elevated` at the same time, so I think you can simplify the selector. Same for other selectors below.

```suggestion
  .v-btn--disabled:not(.v-btn--icon).v-btn--variant-elevated  {
```

**Review by bludnic — CHANGES_REQUESTED** [review]

**Review by bludnic — APPROVED** [review]

**Pull request merged**